supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -238,15 +264,29 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     }
   }
 
+  private def addSystemLogsToStatusAsync(status: TesRunStatus, handle: StandardAsyncPendingExecutionHandle): Future[TesRunStatus] = {
+    makeRequest[Task](HttpRequest(uri = s"$tesEndpoint/${handle.pendingJob.jobId}?view=FULL")) map { response =>
+      status.setStatus(response.state.getOrElse(status.status))
+      status.setLogs(response.logs.last(0).system_logs.getOrElse(Seq.empty[String]))
```

Comment:
@jgainerdewar Clarification: `... logs.flatMap(_.headOption) ...` will get us just the last entry in `logs`, or will get us everything in all entries of `logs`?

Reply:
It will get the system log of the last entry of the `TaskLog`. Or, it will if you use `lastOption` instead of `headOption` like I did... The code above actually gets the system log of the first entry.

This might make it clearer:
```
val logsOpt: Option[Seq[TaskLog]] = response.logs
val finalLogOpt: Option[TaskLog]] = logs.flatMap(_.lastOption)
val sysLogOpt: Option[Seq[String]] = finalLog.flatMap(_.system_logs)
val sysLog: Seq[String] = sysLogOpt.getOrElse(Seq.empty[String])
```